pkg/apiserver/apiserver.go

```diff
@@ -149,7 +149,8 @@ func (s *Server) clusters(w http.ResponseWriter, req *http.Request) {
 	)
 
 	if matches := util.FindNamedStringSubmatch(clusterStatusURL, req.URL.Path); matches != nil {
-		resp, err = s.controller.ClusterStatus(matches["team"], matches["cluster"])
+		namespace, _ := matches["namespace"]
```

Comment:
Does a separate var for `namespace` exist because the namespace is undefined  in some requests  and we need to ignore the error in such cases ?